test/unit/reductions/moments/test_moments_demographic_parity.py

```diff
@@ -140,8 +140,8 @@ def test_signed_weights():
     a0_threshold = 0.2
     a1_threshold = 0.7
 
-    a0_label = "OneThing"
-    a1_label = "AnotherThing"
+    a0_label = 0xDEAD
```

Comment:
What's up with these? :O

Reply:
The moving around of the input conversions has resulted in a requirement for numerical labels. As for the specific values themselves.... `0xDEADBEEF` was used a a 'fill' value for memory, to make it easier to catch uninitialised access:
https://en.wikipedia.org/wiki/Magic_number_(programming)#Magic_debug_values